src/elements/form/NovoFormControl.ts

```diff
@@ -51,27 +51,44 @@ export class NovoFormControl extends FormControl {
      */
     public setRequired(isRequired: boolean): void {
         this.required = isRequired;
-        let validators: any = [...this.validators];
         // Update validators to have the required
         if (this.required && !this.hasRequiredValidator) {
+            let validators: any = [...this.validators];
```

Comment:
It was in a previous commit. I DRY'd out the code and cause a bug because there's a case in between the if and the else that's needed for preventing fieldInteractions on NovoControls from being circular when they're set to fire onInit and onChange.